test/parallel/test-tls-0-dns-altname.js

```diff
@@ -2,6 +2,8 @@
 const common = require('../common');
 const assert = require('assert');
 
+// check getPeerCertificate can properly handle '\0' for fix CVE-2009-2408
```

Comment:
Good to have a test description, it should be a capitalized and period terminated sentence.